addons/l10n_be/i18n/de.po

```diff
@@ -264,36 +205,22 @@ msgstr "Kontenplanvorlage"
 
 #. module: l10n_be
 #. odoo-python
-#: code:addons/l10n_be/models/template_be_ass.py:0
+#: code:addons/l10n_be/models/template_be_asso.py:0
 #, python-format
 msgid "Associations and Foundations"
-msgstr "Vereinigungen und Stiftungen"
+msgstr ""
 
 #. module: l10n_be
 #: model:account.report.column,name:l10n_be.tax_report_vat_balance
 msgid "Balance"
 msgstr "Saldo"
 
-#. module: l10n_be
-#. odoo-python
-#: code:addons/l10n_be/models/template_be.py:0
-#, python-format
-msgid "Bank Fees (21% VAT)"
-msgstr "Bankgebühren (21 % MwSt.)"
-
-#. module: l10n_be
-#. odoo-python
-#: code:addons/l10n_be/models/template_be.py:0
-#, python-format
-msgid "Bank Fees (No VAT)"
-msgstr "Bankgebühren (ohne MwSt.)"
-
```

Comment:
Can't we translate this anymore? Or did it never work? Not sure.

Reply:
The po files have been updated based on the freshly re-exported pot file. If it was deleted, it means that these source terms didn't exist anymore